# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/wasm/WasmCode.h
# Commit: e71e54d93af8
# Full Hash: e71e54d93af8baa6ef915c81dc04ac3400f825cd
# Author: Andy Wingo <wingo@igalia.com>
# Date: 2020-03-06 20:10:41
# Regressor Bug: 1620197
# File Overlap Count: 1
# Description:
#   Bug 1620197 - Enable multiple results from WebAssembly functions r=lth
#   
#   This patch enables multi-value calls and returns, adding some tests, and
#   conditionally disabling a couple expect-fail reftests that now pass.
#   
# ==============================================================================

diff -r afbf73fb4073 -r e71e54d93af8 js/src/wasm/WasmCode.h
--- a/js/src/wasm/WasmCode.h	Fri Mar 06 09:57:45 2020 +0000
+++ b/js/src/wasm/WasmCode.h	Fri Mar 06 13:53:07 2020 +0000
@@ -239,6 +239,7 @@
 
   bool canHaveJitEntry() const {
     return !funcType_.temporarilyUnsupportedReftypeForEntry() &&
+           !funcType_.temporarilyUnsupportedResultCountForEntry() &&
            JitOptions.enableWasmJitEntry;
   }
 